cifar10-multiannotator/multiannotator_labels.ipynb

```diff
@@ -0,0 +1,378 @@
+    "    consensus_labels_out = model_results[i]['consensus_labels_out']\n",
+    "\n",
+    "    acc_model_gtruth = (pred_probs.argmax(axis=1) == true_labels).mean()\n",
+    "    acc_consensus_gtruth = (consensus_labels_in == true_labels).mean()\n",
+    "\n",
+    "    results = {\n",
+    "        \"consensus_gtruth_accuracy\": acc_consensus_gtruth, # consensus labels accuracy \n",
+    "        \"model_gtruth_accuracy\": acc_model_gtruth,         # model label accuracy\n",
+    "    }\n",
+    "    \n",
+    "    model_results[i].update(results)\n",
+    "\n",
+    "# Calculate accuracy of final CROWDLAB generated consensus label\n",
+    "final_consensus_labels = model_results[itter-1][\"consensus_labels_out\"]\n",
+    "final_consensus_accuracy = (final_consensus_labels == true_labels).mean()\n",
+    "\n",
+    "print(\"Final consensus label accuracy (vs ground truth): \", final_consensus_accuracy)\n",
+    "print(\"Final model predictions accuracy (vs ground truth): \", model_results[itter-1][\"model_gtruth_accuracy\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03bb6b74",
+   "metadata": {},
+   "source": [
+    "Finally lets plot the results over multiple epochs and observe the improvement. Here epoch -1 corresponds to the initial `consensus_labels` generated using majority vote without the help of a model. Generating consensus with majority vote is the most common practice and therefore makes a great baseline to compare CROWDLAB against.\n",
+    "\n",
+    "As you can see, the accuracy of the `consensus_labels` improves over several itterations of model training."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0b97270d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "consensus_gtruth_accuracy = [model_results[i][\"consensus_gtruth_accuracy\"] \n",
+    "                             for i in range(itter)] + [final_consensus_accuracy]\n",
+    "\n",
+    "model_gtruth_accuracy = [model_results[i][\"model_gtruth_accuracy\"] for i in range(itter)]\n",
+    "\n",
+    "\n",
+    "# plot prc\n",
+    "plt.rcParams[\"figure.figsize\"] = (17,6)\n",
+    "plt.show()\n",
+    "\n",
+    "plt.subplot(1, 2, 1)\n",
+    "plt.plot(range(-1, itter),consensus_gtruth_accuracy)\n",
+    "plt.xlabel(\"Epochs\", fontsize=14)\n",
+    "plt.ylabel(\"Consensus Label Accuracy\", fontsize=14)\n",
+    "plt.title(\"Consensus Label vs Ground Truth Accuracy\", fontsize=14, fontweight=\"bold\")\n",
+    "\n",
+    "plt.subplot(1, 2, 2)\n",
+    "plt.plot(range(0, itter), model_gtruth_accuracy)\n",
+    "plt.xlabel(\"Epochs\", fontsize=14)\n",
+    "plt.ylabel(\"Model Prediction Accuracy\", fontsize=14)\n",
+    "plt.title(\"Model Predictions vs Ground Truth Accuracy\", fontsize=14, fontweight=\"bold\")\n",
+    "\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aeac8b61",
+   "metadata": {
+    "nbsphinx": "hidden"
```

Comment:
Note anything related to nbsphinx has no effect for `examples/` notebooks since we have no `nbsphinx` running (or any CI for this repo really).

For the notebooks in this repo, the .ipynb file is the only item the user interacts with, as well as just looking at the .ipynb file in github (which is why it is good if the cells are all already executed with their outputs shown)

Reply:
Given that, I'm not sure how I feel about such "unit-test" cells in these notebooks (especially raising an exception seems weird to user). They are definitely good to have in the future though... 

So perhaps a good compromise is to have this cell commented out entirely with disclaimer that it is only to be executed during CI (even though there is no CI running at the moment).   
What do you think?